otx/algorithms/detection/configs/detection/mobilenetv2_ssd/model.py

```diff
@@ -23,12 +23,18 @@
 
 __width_mult = 1.0
 
+runner = dict(max_epochs=20)
```

Comment:
Where did this come from?

Reply:
It came from nowhere. This is caused by the change `CustomSSDHead` in [`this commit`](https://github.com/openvinotoolkit/model_preparation_algorithm/commit/3d128fe720a576cec229eb94b9c1c13b625fdd9a).
Although the architectures of old SSDHead and new SSDHead are the same, there are subtle differences when initializing layers which make [`load_sate_dict_pre_hook()`](https://github.com/openvinotoolkit/model_preparation_algorithm/blob/develop/mpa/modules/models/detectors/custom_single_stage_detector.py#L68-L126) go wrong.
I brought back the old initialization in SSDHead in https://github.com/openvinotoolkit/training_extensions/pull/1316/commits/f8c7cbade82d6967021af47c666d5c1abd5b68e4